lib/ui/text.dart

```diff
@@ -763,11 +763,16 @@ abstract class Paragraph extends NativeFieldWrapperClass2 {
   /// Word boundaries are defined more precisely in Unicode Standard Annex #29
   /// http://www.unicode.org/reports/tr29/#Word_Boundaries
   List<int> getWordBoundary(int offset) native "Paragraph_getWordBoundary";
+  
+  // Toggles between Blink text shaping backend and libtxt backend. This allows
+  // for debugging and development and is not intended to be a final feature.
+  void toggleTxt() native "Paragraph_toggleTxt";
```

Comment:
I have removed this function for now and started with the flag. I'll finish up the command line flag in a later patch. This pr is getting really big!